docs/CONFIG-VARS.md

```diff
@@ -295,8 +300,8 @@ Each server element, like `foo = {}`, can contain none, some, or all of the para
 | multi_az | Specifies if PostgreSQL instance is multi-AZ | bool | false | |
 | deletion_protection | Protect from accidental resource deletion | bool | false | |
 | ssl_enforcement_enabled | Enforce SSL on connections to PostgreSQL server instance | bool | true | |
-| parameters | additional parameters for PostgreSQL server | list(map(string)) | [] | More details can be found [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Appendix.PostgreSQL.CommonDBATasks.html#Appendix.PostgreSQL.CommonDBATasks.Parameters) |
```

Comment:
This code should be in place not sure how your branched missed these. Pull 298 and 299 back into the file.